controller/iteration_blackbox_test.go

```diff
@@ -399,7 +445,11 @@ func (rest *TestIterationREST) TestFailShowIterationMissing() {
 
 func (rest *TestIterationREST) TestSuccessUpdateIteration() {
```

Comment:
more or less same comment as @kwk, we should just have a `TestSuccess` method here, with nested tests for all cases (`ok`, ..)

Reply:
yes I have tried doing the same here, https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/80ae86445281239f604d0a038eba40ff8b347607
thanks

Reply:
ok, thanks